install.rb

```diff
@@ -299,11 +299,17 @@ def prepare_installation
   bindir = join(destdir, bindir)
   mandir = join(destdir, mandir)
   sitelibdir = join(destdir, sitelibdir)
+  vardir = "/opt/puppetlabs/agent/cache"
+  rundir = "/var/run/puppetlabs/agent"
+  logdir = "/var/log/puppetlabs/agent"
 
   FileUtils.makedirs(configdir) if InstallOptions.configs
   FileUtils.makedirs(bindir)
   FileUtils.makedirs(mandir)
   FileUtils.makedirs(sitelibdir)
+  FileUtils.makedirs(vardir)
+  FileUtils.makedirs(rundir)
+  FileUtils.makedirs(logdir)
```

Comment:
Ah I was still thinking `rundir` and `logdir` were still under `vardir`, but of course they're not anymore. And that's correct that puppet won't create parent directories, so :+1: